onebusaway-android/src/main/java/org/onebusaway/android/tripservice/NotifierTask.java

```diff
@@ -161,23 +165,34 @@ private Notification createNotification(String routeId,
                 .setContentIntent(contentIntent)
                 .setDeleteIntent(deleteIntent)
                 .setContentTitle(title)
-                .setContentText(getNotifyText(routeId, timeDiff))
+                .setContentText(getNotifyText(routeId, timeDiff, isArriving))
                 .build();
 
     }
 
-    private String getNotifyText(String routeId, long timeDiff) {
+    private String getNotifyText(String routeId, long timeDiff, boolean isArriving) {
```

Comment:
The more I think about this, the more I think the contents of this method should be refactored into `ArrivalInfo` as part of the logic in the constructor.  It looks a lot like the code to generate the status and time labels there, and we should try to keep all the code that generates text from arrival/departure info in one place - we will likely use the reminder text elsewhere (e.g., notifying someone when they approach a stop - https://github.com/OneBusAway/onebusaway-android/issues/184).  You might actually be able to generate the same text from the status text IF/THEN block.  Could you please refactor this code there, and add a few unit test asserts to `UIUtilTest.testArrivalInfoLabels()`?